modules/abstract-utxo/src/abstractUtxoCoin.ts

```diff
@@ -1442,12 +1442,32 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
     };
   }
 
-  async getExtraPrebuildParams(buildParams: ExtraPrebuildParamsOptions & { wallet: Wallet }): Promise<any> {
-    if (buildParams.wallet.subType() === 'distributedCustody') {
-      return { txFormat: 'psbt' };
+  async getExtraPrebuildParams(buildParams: ExtraPrebuildParamsOptions & { wallet: Wallet }): Promise<{
+    txFormat?: 'legacy' | 'psbt';
+    addressType?: ScriptType2Of3;
+  }> {
+    let txFormat, addressType;
+
+    if (
+      buildParams.txFormat === undefined &&
+      (buildParams.wallet.subType() === 'distributedCustody' ||
+        (isTestnet(this.network) && buildParams.wallet.type() === 'hot'))
+    ) {
+      txFormat = 'psbt';
     }
 
-    return {};
+    if (
+      buildParams.addressType === undefined &&
+      buildParams.wallet.type() === 'hot' &&
+      this.network === utxolib.networks.testnet
```

Comment:
```suggestion
      isTestnet(this.network)
```

Reply:
`isTestnet` checks for any UTXO `testnet` network. `this.network === utxolib.networks.testnet` checks for BTC testnet. We require the latter.